src/txacme/testing.py

```diff
@@ -19,6 +21,24 @@
 from txacme.util import clock_now, generate_private_key
 
 
+class TXACMETestCase(TestCase):
+    """
+    Common code for all tests for the txacme project.
```

Comment:
I definitely don't want all the tests in the whole project to have special nonstandard behavior :)